grafanaplane/oncall/escalationchain.libsonnet

```diff
@@ -36,12 +36,15 @@ local escalation = raw.oncall.v1alpha1.escalation;
     ]
   ),
   withSteps(steps):: {
+    local this = self,
     steps:
       std.mapWithIndex(
         function(position, step)
-          local id = '%s-%d' % [self.chainName, position];
+          local id = '%s-%d' % [this.chainName, position];
           escalation.new(id)
-          + escalation.spec.parameters.forProvider.escalationChainRef.withName(self.chainName)
+          + escalation.spec.parameters.forProvider.escalationChainSelector.withMatchLabels({
+            'crossplane.io/claim-name': this.chainName,
```

Comment:
Theoretically, the same claim-name could exist in a different namespace, so this is probably not sufficient in the long run.

Reply:
I think this is a downside of keeping all the logic in jsonnet as opposed to using Crossplane Compositions.

Reply:
Why doesn't/won't `escalationChainRef.withName()` work?